pkg/trace/sampler/remote_config_handler.go

```diff
@@ -0,0 +1,73 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package sampler
```

Comment:
I think `APMRemoteConfigHandler` shouldn't be in package `sampler`. It has nothing to do with samplers.
I think we also should be able to push non sampler configs.

Reply:
I agree, do you see an obvious place for it?

Otherwise I think it's fine to refactor in the end when we're sure everything is there and working